iron-gpg.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016 IronCore Labs <bob.wall@ironcorelabs.com>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef _IRON_GPG_H
+#define _IRON_GPG_H
+
+#include "sodium.h"
+#include "sshbuf.h"
+
+
+#define COMMENT_MAX				128		//  Max # bytes in the comment on a public SSH key
+
+#define GPG_MAX_KEY_SIZE		512		//  # bytes a secret or public key can occupy
```

Comment:
This seems really big for Curve25519, which would be 32. Is this including other things? Or allowing for RSA? If RSA, it is pretty small.